pkg/workload/admissionchecks.go

```diff
@@ -44,6 +45,10 @@ func SyncAdmittedCondition(w *kueue.Workload) bool {
 		Message: "The workload is admitted",
 	}
 	switch {
+	case !hasReservation && !isActive:
+		newCondition.Status = metav1.ConditionFalse
+		newCondition.Reason = "QueueingDisabled"
+		newCondition.Message = "Active is set to false"
```

Comment:
I don't think this will be necessary.

syncAdmission is just about QuotaReserved and AdmissionChecks. We don't handle evictions here.